Mathlib/CategoryTheory/Limits/Shapes/SequentialProduct.lean

```diff
@@ -71,9 +71,10 @@ lemma functorMap_commSq_aux {n m k : ℕ} (h : n ≤ m) (hh : ¬(k < m)) :
       eqToHom (functorObj_eq_neg (by omega : ¬(k < n))) =
         (Pi.π (fun i ↦ if _ : i < m then M i else N i) k) ≫
           eqToHom (functorObj_eq_neg hh) := by
-  induction' h using Nat.leRec with m h ih
-  · simp
-  · specialize ih (by omega)
+  induction h using Nat.leRec with
+  | refl => simp
+  | @le_succ_of_le m h ih =>
```

Comment:
Where is `leRec` defined? Should we change the explicitness?

Reply:
`leRec` is defined in **core**:
```lean
@[elab_as_elim]
def leRec {n} {motive : (m : ℕ) → n ≤ m → Sort*}
    (refl : motive n (Nat.le_refl _))
    (le_succ_of_le : ∀ ⦃k⦄ (h : n ≤ k), motive k h → motive (k + 1) (le_succ_of_le h)) :
    ∀ {m} (h : n ≤ m), motive m h
  | 0, H => Nat.eq_zero_of_le_zero H ▸ refl
  | m + 1, H =>
    (le_succ_iff.1 H).by_cases
      (fun h : n ≤ m ↦ le_succ_of_le h <| leRec refl le_succ_of_le h)
      (fun h : n = m + 1 ↦ h ▸ refl)
```
I think the strict-implicit binder should be changed to a plain explicit binder.